src/ci/github-actions/jobs.yml

```diff
@@ -212,11 +212,62 @@ auto:
   - image: dist-x86_64-netbsd
     <<: *job-linux-4c
 
-  - image: i686-gnu
-    <<: *job-linux-8c
+  # The i686-gnu job is split into multiple jobs to run tests in parallel.
+  #
+  # The `src/*` and `tests/*` tests are skipped to speed up this slow job.
+  # These tests can be skipped because they are unlikely to be platform specific.
```

Comment:
Do these `src` and `rustdoc` tests take so much time? These skips are kind of random. Running bootstrap tests on different platforms would be nice, even though it currently has low coverage.